src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type CommentsInfos = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<CommentsInfos> = ({ comment }) => (
```

Comment:
de-structuring would save you 4x `comment.`
```suggestion
export const CommentInfo: React.FC<CommentsInfos> = ({ {name, email, body} }) => (
```

Reply:
Done! thanks Roman!